### DISCUSSION TITLE
Download the built package

### DISCUSSION THREAD

**Discussion by MPloier** [dsoftwareinc/ghactions-manager#103] [Ideas]
Hi guys,
I wondered, if its possible that you can add a button, which downloads the package of the highlighted task.
Maybe on the left side under the buttons "Refresh Workflow Runs" and "Open GHActions-Manager Settings".
I would try it myself but I never programed Kotlin sooo yeah. 🫣
Just a idea :D 

<img width="746" alt="image" src="https://github.com/cunla/ghactions-manager/assets/117640979/e2521bb9-181e-4c1d-a41b-03f5befb3c3f">

**Comment by cunla** [comment]
Hi, I am not sure about this - it would mean analyze the github workflow and understand that there is a package published. That is not really the scope of "managing github-actions" and quite specific.

You are welcome to implement it if you wish - I would add it as an action on the popup menu of the workflow-run - that way it can be added only when relevant for the workflow.

**Reply by MPloier** [reply]
Im sorry I was not that precise. I meant an additional button below those two buttons "Refresh Workflow Runs" and "Open GHActions-Manager Settings". Just a download button that is disabled when there is no package to download.

**Reply by cunla** [reply]
I understood what you meant. Since it applies to a workflow and not to the plugin, I think it should be in the pop-up menu.
